crates/bevy_render/src/camera/clear_color.rs

```diff
@@ -3,11 +3,15 @@ use bevy_color::Color;
 use bevy_derive::{Deref, DerefMut};
 use bevy_ecs::prelude::*;
 use bevy_reflect::prelude::*;
-use serde::{Deserialize, Serialize};
 
 /// For a camera, specifies the color used to clear the viewport before rendering.
-#[derive(Reflect, Serialize, Deserialize, Clone, Debug, Default)]
-#[reflect(Serialize, Deserialize, Default)]
+#[derive(Reflect, Clone, Debug, Default)]
+#[reflect(Default)]
+#[cfg_attr(
+    feature = "serialize",
+    derive(serde::Serialize, serde::Deserialize),
+    reflect(Serialize, Deserialize)
+)]
```

Comment:
This will not go right. I'm currently working in making bevy_render serialize feature. For now, can you remove this and do the import of bevy_color on bevy_render TOML import with the serialize feature? Something like this will do:

```toml
bevy_color = { version = /* . . . */, features = ["serialize"] }
```

Reply:
You mean restoring to: 

#[derive(Reflect, Serialize, Deserialize, Clone, Debug, Default)]
#[reflect(Serialize, Deserialize, Default)]

Reply:
Yep. Instead, we will use the feature on in bevy_color by default and avoid the compile error.